Gemfile

```diff
@@ -22,7 +22,7 @@ end
 # as a dependency of govuk_content_models
 gem 'mongo', '>= 1.7.1'
 
-gem 'gds-sso', '10.0.0'
+gem 'gds-sso', '11.0.0'
```

Comment:
As there's uncertainty here, and seeing as this story is just about upgrading to Ruby 2.1.6, I think it's worth bringing gds-sso back down to a `10.x` version here